pkg/storage/sds/edvformatprovider/edvformatprovider.go

```diff
@@ -0,0 +1,183 @@
+/*
+Copyright SecureKey Technologies Inc. All Rights Reserved.
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package edvformatprovider
```

Comment:
should it just be edvprovider - why format?

Reply:
As discussed, names it just `FormatProvider` for now.